libtiledbsoma/src/utils/column_buffer.cc

```diff
@@ -56,7 +56,7 @@ std::shared_ptr<ColumnBuffer> ColumnBuffer::create(
         if (enum_name.has_value()) {
             enumeration = std::make_optional<Enumeration>(
                 ArrayExperimental::get_enumeration(
-                    schema.context(), *array, name_str));
+                    schema.context(), *array, *enum_name));
```

Comment:
Nice.  That may have been what I ran into earlier flipping between my branch and your branch and trying an example with 'multiple column using one enumeration'.  Will try this tomorrow.

Reply:
I did indeed do the trick.